README.md

```diff
@@ -11,10 +11,19 @@ The SPS frontend is a lightweight implementation of Epic Games' [Pixel Streaming
 
 ## Documentation
 
-In general, the official Epic Games Pixel Streaming [frontend docs](https://github.com/EpicGames/PixelStreamingInfrastructure/tree/master/Frontend) should cover most common usage cases (as our library is simply a thin wrapper on that). However, some SPS frontend specific docs are listed below:
+### Utilising the Scalable Pixel Streaming Frontend
+Below for a comprehensive guide to accessing the Scalable Pixel Streaming Frontend, including where to download the library and how to consume and customise it for your own projects.
```

Comment:
Perhaps should start with "Follow the link below"?

Reply:
This has been restructured to flow better